src/server/qgsserver.h

```diff
@@ -117,7 +117,7 @@ class SERVER_EXPORT QgsServer
     // Status
     static QString* sConfigFilePath;
     static QgsCapabilitiesCache* sCapabilitiesCache;
-    static QgsMapRenderer* sMapRenderer;
+    static QgsMapSettings* sMapSettings;
```

Comment:
Is this still used anywhere? Not sure if we need a static map settings instance...